### PR TITLE
Grammar edits to documentation; small additions to make testing more …

### PR DESCRIPTION
**Pull request by juliaputko**
…clear

### CODE REVIEW

File: doc/experiment.rst [hunk 1]
```diff
@@ -182,22 +183,22 @@ workload manager and available compute resources.
...
 
-If passed ``BatchSettings`` and ``RunSettings``, the ``BatchSettings`` will
+If it passed ``BatchSettings`` and ``RunSettings``, the ``BatchSettings`` will
 determine the allocation settings for the entire batch, and the ``RunSettings``
 will determine how each individual ``Model`` instance is executed within
 that batch.
 
-The same example as above but tailored towards a running as a batch job
-on a slurm system
+This is same example as above, but tailored towards a running as a batch job
```
Comment (billschereriii) [inline doc/experiment.rst:200]: Should be "This is `the`"

File: doc/overview.rst [hunk 2]
```diff
@@ -58,19 +58,19 @@ deploying HPC workloads alongside an in-memory database: Redis.
 
 The key features of the IL are:
 
-  - An API to start, monitor, and stop HPC jobs from Python or Jupyter notebook.
+  - An API to start, monitor, and stop HPC jobs from a Python or Jupyter notebook.
```
Comment (billschereriii) [inline doc/overview.rst:61]: change to "from Python or from a Jupyter notebook"

File: doc/experiment.rst (hunk 1 above)
Comment (juliaputko) [reply doc/experiment.rst:200]: Will do!

File: doc/overview.rst (hunk 2 above)
Comment (juliaputko) [reply doc/overview.rst:61]: Will do!

**Pull request merged**